drakrun/drakrun/lib/sample_startup.py

```diff
@@ -39,14 +34,13 @@ def get_office_file_startup_command(extension, file_path):
     elif is_office_powerpoint_file(extension):
         start_command.append("powerpnt.exe")
     else:
-        log.warning(f"Unknown office file extension {extension}.")
-        return None
+        raise RuntimeError(f"Unknown office file extension {extension}.")
     start_command.extend(["/t", "%f"])
 
     if file_type_allows_macros(extension):
-        vbaparser = VBA_Parser(file_path)
+        vbaparser = VBA_Parser(f"malware.{extension}", data=content)
         if vbaparser.detect_vba_macros():
-            outer_macros = get_outer_nodes_from_vba_file(file_path)
+            outer_macros = get_outer_nodes_from_vba_file(vbaparser)
```

Comment:
Is it receiving VBA_Parser instance or file path? I think that changes in vba_graph.py might be missing in this PR.